### PR TITLE
Add RSS3 mainnet and sepolia testnet info

### PR DESCRIPTION
**Pull request by Songkeys**

### CODE REVIEW

**Comment by Songkeys** [comment]
@ligi Hi. I think this could be merged now. Could you please take another look? Thank you!

**Comment by azuic** [comment]
hey @ligi - we’d like to support the maintenance of the repo and hope this donation can help: 0xbb94be3dc177071fb1e06bc0783cfa6661601bed8bb1f0cb5c2aa4d183dfab9b

meanwhile, is it possible to expedite the pr?

File: _data/chains/eip155-2331.json [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+  "nativeCurrency": {
+    "name": "RSS3",
+    "symbol": "RSS3",
+    "decimals": 18
+  },
+  "infoURL": "https://rss3.io",
+  "shortName": "rss3-testnet",
+  "chainId": 2331,
+  "networkId": 2331,
+  "icon": "rss3",
```
Comment (ligi) [inline _data/chains/eip155-2331.json:15]:
```suggestion
  "icon": "rss3-testnet",
```

**Review by ligi — CHANGES_REQUESTED** [review]

**Review by ligi — APPROVED** [review]
Thanks - fixed the error and merging the PR

**Pull request merged**